src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -734,6 +746,15 @@ void Transformations::MainSnippets(void) {
     tokenization_config.concurrency = config.streamExecutorConfig.get_threads_per_stream();
     if (tokenization_config.concurrency == 0)
         tokenization_config.concurrency = parallel_get_max_threads();
+#if defined(OPENVINO_ARCH_ARM64)
+    // ARM has 32 gprs. After excluding 2 registers for work amounts, 1 platform register, 3 registers for temporary use,
+    // and 2 stack related registers, it has 24 remaining registers.
+    tokenization_config.data_ptr_grp_count = 24;
```

Comment:
Probably, there should be 23 since we [remove](https://github.com/openvinotoolkit/openvino/pull/23180#discussion_r1524354300) `runtime parameters` registers from gpr pool.
Could you please check?

Reply:
Applied. Thanks Alexandra!